README.md

```diff
@@ -21,6 +21,7 @@ See the [CONTRIBUTING](./CONTRIBUTING.md)
 | [`@tpr/core`](packages/core)       | [![npm version][core npm version]][core npm link]       | [![Bundle Size][core size bundle]][core size link]       |
 | [`@tpr/forms`](packages/forms)     | [![npm version][forms npm version]][forms npm link]     | [![Bundle Size][forms size bundle]][forms size link]     |
 | [`@tpr/icons`](packages/icons)     | [![npm version][icons npm version]][icons npm link]     | [![Bundle Size][icons size bundle]][icons size link]     |
+| [`@tpr/layout`](packages/layout)   | [![npm version][layout npm version]][layout npm link]   | [![Bundle Size][layout size bundle]][layout size link]   |
```

Comment:
I don't know why but this bit doesn't actually work. If you look at the home page of the repo it's not interpreted like the others.

Reply:
mmm, I know the issue ... somehow some code got deleted for the layout bit in Readme,md